rpcclient/infrastructure.go

```diff
@@ -1698,3 +1725,73 @@ func (c *Client) Send() error {
 
 	return nil
 }
+
+func ParseAddressString(strAddress string, defaultPort string) (net.Addr, error) {
```

Comment:
I believe this looks cleaner now, just a few nitpicks to address. please ensure to add comments for the functions added, as specified in the contribution guide linked [here](https://github.com/btcsuite/btcd/blob/master/docs/code_contribution_guidelines.md?plain=1#L107)

Reply:
Thanks, added the comments